docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc

```diff
@@ -650,6 +650,111 @@ Tables must be created/dropped/altered through Hive itself.
 [[load_examples]]
 === Examples of LOAD
 
+* This example demonstrates the LOAD statement continues loading data from source_table into target_table after ignorable data conversion error.
+
++ 
+Suppose that we have two tables (source_table and target_table) like this:

```

Comment:
remove the 'like this'?
Or 'like these'?
I am not sure :-)

Reply:
Thanks Ming :)
"this" represents the following situation (whole, singular).

Two similar examples from other manuals are attched for your reference:
1. The example in _Oracle SQL Reference Manual_
![like this 1](https://user-images.githubusercontent.com/20532956/34660496-c6dba0a2-f47d-11e7-9b76-6c706e1d07f4.jpg)

2. The example in _MySQL SQL Reference Manual_
![like this 2](https://user-images.githubusercontent.com/20532956/34660502-ccbeb374-f47d-11e7-9f82-4c16166425ea.jpg)

Reply:
you are right ^_^
I learned something new.